### PR TITLE
Fix PRF-6996

### PR DESCRIPTION
**Pull request by vrockz747**
BEP-20 Tokens now display checksum'd address for receiver

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6996 On device in send txn of BEP-20 tokens receiver's address casing is different from cysync](https://app.clickup.com/t/9002019994/PRF-6996)

File: common/coin_support/eth_sign_data/abi_decode.c [hunk 1]
```diff
@@ -215,11 +216,11 @@ ui_display_node *ABI_Stringify(Abi_Type_e inputAbiType,
...
-      byte_array_to_hex_string(pAbiTypeData + Abi_address_e_OFFSET_BE,
-                               20,
-                               &(staticBufferInUTF8[2]),
-                               41);
-
+      // generating checksum'd address
+      ethereum_address_checksum((pAbiTypeData + Abi_address_e_OFFSET_BE),
+                                &(staticBufferInUTF8[2]),
+                                false,
+                                0);
```
Comment (ujjwal-cyph) [inline common/coin_support/eth_sign_data/abi_decode.c:223]: if possible, pass chain id as a parameter.

Comment (vrockz747) [reply common/coin_support/eth_sign_data/abi_decode.c]: added

**Review by ujjwal-cyph — APPROVED** [review]
LGMT

**Pull request merged**